sig-cloud-provider/CHARTER.md

```diff
@@ -0,0 +1,100 @@
+# SIG Cloud Provider Charter
+
+## Mission
+The Cloud Provider SIG ensures that the Kubernetes ecosystem is evolving in a way that is neutral to all (public and private) cloud providers. It will be responsible for establishing standards and requirements that must be met by all providers to ensure optimal integration with Kubernetes.
+
+## Subprojects & Areas of Focus
+
+* Maintaining parts of the Kubernetes project that allows Kubernetes to integrate with the underlying provider. This includes but are not limited to:
+    * [cloud provider interface](https://github.com/kubernetes/kubernetes/blob/master/pkg/cloudprovider/cloud.go)
+    * [cloud-controller-manager](https://github.com/kubernetes/kubernetes/tree/master/cmd/cloud-controller-manager)
+    * Deployment tooling which has historically resided under [cluster/](https://github.com/kubernetes/kubernetes/tree/release-1.11/cluster)
+* Code ownership for all cloud providers that fall under the kubernetes organization and have opted to be subprojects of SIG Cloud Provider. Following the guidelines around subprojects we anticipate providers will have full autonomy to maintain their own repositories, however, official code ownership will still belong to SIG Cloud Provider.
+    * [cloud-provider-azure](https://github.com/kubernetes/cloud-provider-azure)
+    * [cloud-provider-gcp](https://github.com/kubernetes/cloud-provider-gcp)
+    * [cloud-provider-openstack](https://github.com/kubernetes/cloud-provider-openstack)
+    * [cloud-provider-vsphere](https://github.com/kubernetes/cloud-provider-vsphere)
```

Comment:
cc @frapposelli @tpepper

Reply:
@andrewsykim  How about cloud-provider-alicloud for now ? Should we add it here now ?

Reply:
@andrewsykim I am also looking forward to getting the `cloud-provider-alicloud` into kubernetes community too. It will be much easier for users to setup kubernetes on alicloud.

cc @aoxn @denverdino

Reply:
The list of repos in the charter is the existing repos (belonging to existing SIGs) that would immediately fall under sig-cloud-provider ownership. Once the SIG is formed we will have a formal process in place for new all providers :).